Editor/EditorCore/EditorSceneViewPicker.cs

```diff
@@ -32,6 +32,25 @@ static class EditorSceneViewPicker
         // When enabled, a mouse click on an unselected mesh will select both the GameObject and the mesh element picked.
         const bool k_AllowUnselected = true;
 
+        public static void DoMouseHover(SceneSelection selection)
+        {
+            if (selection.faces.Count == 0)
+                return;
+            var mesh = selection.mesh;
+            var face = selection.faces[0];
+            var activeFace = mesh.GetActiveFace();
+            if (activeFace == null || activeFace == face)
+                return;
+
+            var faces = mesh.facesInternal;
+            var pathFaces = SelectPathFaces.GetPath(mesh, Array.IndexOf<Face>(faces, mesh.GetActiveFace()),
+                                 Array.IndexOf<Face>(faces, face));
```

Comment:
You're calling `mesh.GetACtiveFace()` twice here instead of using `activeFace`.

In the interest of saving on allocations, could we add instead an overload of `GetPath` that accepts a `List<Face>` to append to instead of returning a new collection?

Reply:
The `GetPath` method returns a list of int but we need to transform that to a list of faces. Plus, the code uses a Stack instead of a List so this would make the code in GetPath a bit more complex. I don't know if it's worth it?

Reply:
Ok, if it's not going to be a notice-able improvement that's fine.